### PR TITLE
Added FXAA and MSAA support to StandardRenderingPipeline

### PR DESCRIPTION
**Pull request by julien-moreau**

### CODE REVIEW

**Comment by julien-moreau** [comment]
Also, Bloom is not enabled by default

File: dist/preview release/what's new.md [hunk 1]
```diff
@@ -49,6 +49,7 @@
 - All NPM packages have `latest`and `preview` streams [#3055](https://github.com/BabylonJS/Babylon.js/issues/3055) ([RaananW](https://github.com/RaananW))
 - Added New Tools Tab in the inspector (env texture and screenshot tools so far) ([sebavan](http://www.github.com/sebavan))
 - Moved to gulp 4, updated dependencies to latest ([RaananW](https://github.com/RaananW))
+- Added FXAA and MSAA support to the StandardRenderingPipeline ([julien-moreau](https://github.com/julien-moreau))
```
Comment (deltakosh) [inline dist/preview release/what's new.md]: I would see this line in the engine part :)

**Comment by julien-moreau** [comment]
Done :)

File: src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts [hunk 2]
```diff
@@ -32,6 +32,8 @@
 
         public depthOfFieldPostProcess: Nullable<PostProcess> = null;
 
+        public fxaaPostProcess: Nullable<FxaaPostProcess> = null;
```
Comment (deltakosh) [inline src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts]: I think the doc check will fail on this one ;)

Comment (julien-moreau) [reply src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts]: That means everything must be commented in the pipeline :)

File: src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts [hunk 3]
```diff
@@ -219,6 +399,10 @@
             this._volumetricLightStepsCount = count;
         }
 
+        /**
+		 * Specifies the number of samples used for the motion blur effect
+         * Typically in interval [16, 64]
```
Comment (deltakosh) [inline src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts]: Spaces are weird

File: src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts [hunk 4]
```diff
@@ -4,89 +4,228 @@
...
         public brightThreshold: number = 1.0;
 
+        /**
+         * Configures the blur intensity used for surexposed surfaces are highlighted surfaces (light halo)
+         */
         @serialize()
         public blurWidth: number = 512.0;
+        /**
+         * Sets if the blur for surexposed surfaces are highlighted surfaces (light halo) must be only horizontal
+         */
```
Comment (deltakosh) [inline src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts]: Syntax seems wrong:)

File: src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts [hunk 5]
```diff
@@ -4,89 +4,228 @@
...
-
+        /**
+         * Light (spot or directional) used to generate the volumetric lights rays
+         * The source light must have a shadow generate so the pipeline can get its 
+         * depth map
+         */
         public sourceLight: Nullable<SpotLight |  DirectionalLight> = null;
 
+        /**
+         * For eye adaption, represents the minimum lumance the eye can see
```
Comment (deltakosh) [inline src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts]: Adaptation and luminance

File: src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts [hunk 6]
```diff
@@ -4,89 +4,228 @@
...
+        /**
+         * For eye adaptation, represents the increase luminance speed
+         */
         @serialize()
         public hdrIncreaseRate: number = 0.5;
 
+        /**
+         * Lens color texture used by the lens flare effect. Must be set and available in
+         * the documentation
+         */
```
Comment (deltakosh) [inline src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts]: What do you mean for documentation?

File: src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts [hunk 7]
```diff
@@ -104,16 +243,21 @@
...
         private _hdrEnabled: boolean = false;
         private _motionBlurEnabled: boolean = false;
+        private _fxaaEnabled: boolean = false;
 
         private _motionBlurSamples: number = 64.0;
         private _volumetricLightStepsCount: number = 50.0;
+        private _samples: number = 1;
 
+        /**
+		 * Specifies if the bloom pipeline is enabled
```
Comment (deltakosh) [inline src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts]: Spaces are incorrect for some comments

**Comment by sebavan** [comment]
Thanks :-)

**Pull request merged**